### DISCUSSION TITLE
How can I get Jotai DevTools to show up in NextJs preview deploy?

### DISCUSSION THREAD

**Discussion by jdaudier** [jotaijs/jotai-devtools#124] [Q&A]
This seems to be the var Jotai DevTools is using to decide whether to show or not. I believe this var is `true` for preview deployments in NextJS, but it's not showing up. Anyone got ideas to help me out 🙏 ? Thank you!

![image](https://github.com/jotaijs/jotai-devtools/assets/1666031/dc7cee0c-7dc6-4baf-ae1d-2302c35f5bde)

**Comment by arjunvegda** [comment]
Hi @jdaudier 

Unfortunately Jotai DevTools are only available in the development mode. Read more about it here: https://github.com/jotaijs/jotai-devtools/discussions/116

**Reply by jdaudier** [reply]
hi @arjunvegda Isn't `process.env.NODE_ENV !== 'production'` = `true` in preview deployment for NextJs, though?

**Reply by arjunvegda** [reply]
I checked the deploy preview of one of my projects using NextJS. It has `process.env.NODE_ENV === 'production'`  🤔 

Could you place a breakpoint on [this line](https://github.com/jotaijs/jotai-devtools/blob/main/src/DevTools/DevTools.tsx#L138) in your app and see what the value is and help me confirm? 

Alternatively, you could also do `console.log({ env: process.env.NODE_ENV })`.
